README.md

```diff
@@ -35,6 +35,24 @@ ZENDESK_PASSWORD=
 ZENDESK_LOCALE=
 ```
 
+### Configure Salesforce as source
+```
+CONFIGURER=salesforce
+# The base url for Salesforce API (eg. https://<company>.my.salesforce.com)
+SALESFORCE_BASE_URL=
+# Used Salesforce API version (eg. v56.0)
+SALESFORCE_API_VERSION=
+# Username, password, clientId, cliendSecret for API authentication
+SALESFORCE_USERNAME=
+SALESFORCE_PASSWORD=
+SALESFORCE_CLIENT_ID=
+SALESFORCE_CLIENT_SECRET=
+# Used language code (eg. en-us)
+SALESFORCE_LANGUAGE_CODE=
+# Names of articles that contain parts of the content as comma separated values (eg. Question__c,Answer__c)
```

Comment:
`Names of article fields` instead of `Names of articles`?

Reply:
That would be misleading. Salesforce articles can contain lots of different fields, and the actual content can be spread across all of them. This variable is used to determine which article fields should be considered as content fields